src/components/CustomCall.tsx

```diff
@@ -14,8 +14,10 @@
 // You should have received a copy of the GNU General Public License
 // along with Parity Bridges UI.  If not, see <http://www.gnu.org/licenses/>.
 
+import Button from '@material-ui/core/Button';
+import Container from '@material-ui/core/Container';
+import TextField from '@material-ui/core/TextField';
 import React, { useState } from 'react';
```

Comment:
Always as a good practice start with import React.... and all others should follow